app/workers/downstream_publish_worker.rb

```diff
@@ -0,0 +1,93 @@
+class DownstreamPublishWorker
+  attr_reader :content_item_id, :payload_version, :message_queue_update_type, :update_dependencies
+  include Sidekiq::Worker
+  include PerformAsyncInQueue
+
+  HIGH_QUEUE = "downstream_high".freeze
+  LOW_QUEUE = "downstream_low".freeze
+
+  sidekiq_options queue: HIGH_QUEUE
+
+  def perform(args = {})
+    assign_attributes(args.symbolize_keys)
+
+    unless web_content_item
+      raise CommandError.new(
+        code: 404,
+        message: "The content item for id: #{content_item_id} was not found",
+      )
+    end
+
+    unless content_item_state?(:published)
+      raise CommandError.new(
+        code: 500,
+        message: "Will not downstream publish a content item that isn't published",
+      )
```

Comment:
I think in this case, the worker should just exit immediately, without raising any error, as the state not being published is a sign that its no longer appropriate to continue, and does not definitely mean that something has gone wrong.